tools/packaging/common/sidecar.env

```diff
@@ -102,3 +101,5 @@ OUTPUT_CERTS=/etc/certs
 # and returning workload certificates. Istiod is implementing the protocol, and is the default value
 # if ISTIO_CA is not set.
 #ISTIO_CA
+# set CA_ADDR if your istiod.istio-system.svc is on port other than 15102
```

Comment:
15102->15012

Reply:
A typo here :( Thanks for catching this up.

Reply:
fixed